ranker/routes/api.py

```diff
@@ -119,7 +119,8 @@ def change_main():
         # Get the correct objects
         player = Player.get_player(content['uid'])
         if player is None:
-            return make_response(jsonify(message="No such player exists"), 400)
+            new_player(content['uid'])
+            player = Player.get_player(content['uid'])
```

Comment:
merge these two lines.  `new_player(uid)` returns the new player object.  There is no need to do another query for the new user.  `player = new_player(...)` is sufficient here